email/marketing-email.html

```diff
@@ -1,17 +1,17 @@
 <!--
-    The proper way to do marketing email with TinyMCE is to use
-    inline mode, and move all table layout outside of TinyMCE.
-    Tables in TinyMCE are first and formost to display tabular
-    data, not for layouts do doing table layouts in TinyMCE,
-    while possible, is not recommended. It is simply too easy
-    to accidentally break the design.
-
-    Here is an example of how to do a marketing email using
-    TinyMCE inline mode. The TinyMCE editor is only applied
-    to the rich text parts. All the table layout are outside
-    of TinyMCE. This makes it easy for you to to add other
-    logic such as inserting blocks and drag and drop reordering
-    yourself.
+The proper way to do marketing email with TinyMCE is to use
+inline mode, and move all table layout outside of TinyMCE.
+Tables in TinyMCE are first and formost to display tabular
+data, not for layouts do doing table layouts in TinyMCE,
+while possible, is not recommended. It is simply too easy
+to accidentally break the design.
+
+Here is an example of how to do a marketing email using
+TinyMCE inline mode. The TinyMCE editor is only applied
+to the rich text parts. All the table layout are outside
+of TinyMCE. This makes it easy for you to to add other
+logic such as inserting blocks and drag and drop reordering
+yourself.
```

Comment:
```suggestion
The proper way to do marketing emails with TinyMCE is to use
inline mode, and move all table layouts outside of TinyMCE.
Tables in TinyMCE are first and foremost to display tabular
data, not for layouts. Doing table layouts in TinyMCE,
while possible, is not recommended. It is simply too easy
to accidentally break the design.

Here is an example of how to do a marketing email using
TinyMCE inline mode. The TinyMCE editor is only applied
to the rich text parts. The table layout is outside
of TinyMCE. This makes it easy for you to to add other
logic such as inserting blocks and drag and drop reordering
yourself.
```
nitpicking the wording